### ISSUE TITLE
I can't open zim file

### ISSUE THREAD

**Issue by carmenfdezb** [mkiol/Zimpedia#33]
I've updated zim file to last version from https://dumps.wikimedia.org/other/kiwix/zim/wikipedia/wikipedia_es_all_maxi_2020-07.zim, but I can't open with Zimpedia.
I get next error if I open Zimpedia from Terminal:

```
nemo ~ $ harbour-zimpedia
[D] unknown:0 - Using Wayland-EGL
[D] unknown:0 - Device pixel ratio: 1.75
[W] unknown:0 - "Couldn't load translation for locale C from /usr/share/harbour-zimpedia/translations"
[D] unknown:0 - Scanning file: "/media/sdcard/TarjetaSD/Diccionario/wikipedia_es_all_maxi_2020-07.zim"

Assertion failed at ../src/buffer.h:42
 size_.v[25971181513] < SIZE_MAX[4294967295]
/usr/share/harbour-zimpedia/lib/libzim.so.4(_Z15_on_assert_failIyjEvPKcS1_S1_T_T0_S1_i+0xfd) [0xf1d40dfa]
/usr/share/harbour-zimpedia/lib/libzim.so.4(_ZN3zim6BufferC1ENS_7zsize_tE+0x81) [0xf1d406c2]
/usr/share/harbour-zimpedia/lib/libzim.so.4(_ZN3zim10MMapBufferC2EiNS_8offset_tENS_7zsize_tE+0x1f) [0xf1d4b8b8]
/usr/share/harbour-zimpedia/lib/libzim.so.4(_ZNK3zim10FileReader10get_bufferENS_8offset_tENS_7zsize_tE+0x1dd) [0xf1d4938a]
/usr/share/harbour-zimpedia/lib/libzim.so.4(_ZN3zim8FileImplC2ERKSs+0xa99) [0xf1d3e776]
/usr/share/harbour-zimpedia/lib/libzim.so.4(_ZN3zim4FileC2ERKSs+0x1d) [0xf1d3b17e]
harbour-zimpedia(+0x1e83c) [0x436683c]
harbour-zimpedia(+0x208da) [0x43688da]
harbour-zimpedia(+0x20668) [0x4368668]
harbour-zimpedia(+0x20668) [0x4368668]
harbour-zimpedia(+0x21184) [0x4369184]
harbour-zimpedia(+0x293ee) [0x43713ee]
/usr/lib/libQt5Core.so.5(+0x8f27e) [0xef51727e]
/lib/libpthread.so.0(+0x6e6a) [0xef451e6a]
[W] unknown:0 - QMutex: destroying locked mutex
QObject::connect: invalid null parameter
Segmentation fault 
```

This file works fine in my PC using Kiwix, so I guess this issue is because of size of zim file. Is that the case? If nothing can be done I will use old zim file.

Thanks for your attention!

**Comment by mkiol** [comment]
Thank you for letting me know about this issue. 

Most likely libzim library is outdated. Could you please test this release: https://github.com/mkiol/Zimpedia/tree/master/binary. I've updated libzim to most recent version. Hopefully it resolves the problem.

**Comment by carmenfdezb** [comment]
Thank you for your quick answer!! I've tested the rpm with libzim updated and it solved my problem!!
Thank you so much!

**Comment by mkiol** [comment]
I'm glad I could help :)

Fix: 110ef01a1004515ab7de62e76c67e5ad25ecd1be

New 2.2.7 release is now on OpenRepos.net
